provider/kubernetes.go

```diff
@@ -125,9 +125,15 @@ func (provider *Kubernetes) loadIngresses(k8sClient k8s.Client) (*types.Configur
 					}
 				}
 				if len(r.Host) > 0 {
+					rule := "Host:" + r.Host
+
+					if strings.Contains(r.Host, "*") {
+						rule = "HostRegexp:" + strings.Replace(r.Host, "*", "{subdomain:[a-z]+}", 1)
```

Comment:
@dtomcej I've changed it to allow dashes and capital letter, but I believe underscores are disallowed in domain names, aren't they?